lmdeploy/model.py

```diff
@@ -1127,11 +1127,13 @@ def __init__(self,
                  eoh='<end_of_turn>\n',
                  assistant='<start_of_turn>model\n',
                  eoa='<end_of_turn>\n',
+                 stop_words=['<end_of_turn>'],
```

Comment:
Does Gemma use the stop_words too?

Reply:
Yes, I guess. Generate eos should means stop.